jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1113,6 +1113,29 @@ def func1(*jax_args):
 
 tf_impl[lax.scan_p] = _scan
 
+def _top_k(operand: TfVal, k: int) -> Tuple[TfVal, TfVal]:
+  # Some types originally incompatible with tf.math.top_k can be promoted
+  # to a compatible type without loss of precision.
+  def promote_dtype(dtype):
```

Comment:
It seems that `dtype` here is a TF dtype (result of `operand.dtype`). If that is true, then it would be clearer if we compare `dtype` with `numpy` types, not `jax.numpy` types (e.g., `dtype in [np.bool_, np.uint8, ...]`)

Reply:
Addressed in latest commits.